pulsar-broker/src/main/java/org/apache/pulsar/broker/rest/Topics.java

```diff
@@ -157,4 +164,140 @@ public void produceOnNonPersistentTopicPartition(@Suspended final AsyncResponse
         }
     }
 
+    @POST
+    @Path("/persistent/{tenant}/{namespace}/{topic}/subscription/{subscription}")
```

Comment:
Shouldn't the API path contain end with "/consumers" if this API method creates a consumer?
```suggestion
    @Path("/persistent/{tenant}/{namespace}/{topic}/subscription/{subscription}/consumers")
```